src/main/resources/templates/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html xmlns:th = "http://www.thymeleaf.org">
+<html lang="en" xmlns:th = "http://www.thymeleaf.org">
```

Comment:
added missing lang attribute to html files it was missing from

ref: https://www.w3.org/International/questions/qa-html-language-declarations